presets/workspace/models/llama3/model.go

```diff
@@ -93,7 +93,7 @@ type llama3_3_70Binstruct struct{}
 func (*llama3_3_70Binstruct) GetInferenceParameters() *model.PresetParam {
 	return &model.PresetParam{
 		Metadata:                  metadata.MustGet(PresetLlama3_3_70BInstructModel),
-		DiskStorageRequirement:    "140Gi",
```

Comment:
220GB = 140G(model weights) + base image( 10GB) + other container in the nodes (overhead)